I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 482, in script
    pov "That your reputation at this school is going to be ruined because I’m going to reveal to the school you’re a [insert words here]."
KeyError: u'insert words here'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 482, in script
    pov "That your reputation at this school is going to be ruined because I’m going to reveal to the school you’re a [insert words here]."
  File "C:\Users\501508\Desktop\renpy-6.99.8-sdk\renpy-6.99.8-sdk\renpy\ast.py", line 603, in execute
    renpy.exports.say(who, what, interact=self.interact)
  File "C:\Users\501508\Desktop\renpy-6.99.8-sdk\renpy-6.99.8-sdk\renpy\exports.py", line 1121, in say
    who(what, interact=interact)
  File "C:\Users\501508\Desktop\renpy-6.99.8-sdk\renpy-6.99.8-sdk\renpy\character.py", line 819, in __call__
    what = what_pattern.replace("[what]", sub(what, translate=translate))
  File "C:\Users\501508\Desktop\renpy-6.99.8-sdk\renpy-6.99.8-sdk\renpy\character.py", line 801, in sub
    return renpy.substitutions.substitute(s, scope=scope, force=force, translate=translate)[0]
  File "C:\Users\501508\Desktop\renpy-6.99.8-sdk\renpy-6.99.8-sdk\renpy\substitutions.py", line 229, in substitute
    s = formatter.vformat(s, (), kwargs)
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 563, in vformat
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 585, in _vformat
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 646, in get_field
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 605, in get_value
KeyError: u'insert words here'

Windows-7-6.1.7601-SP1
Ren'Py 6.99.8.959
Homecoming 0.0
